eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -193,9 +193,9 @@ def clinvar_to_evidence_strings(allowed_clinical_significance, mappings, json_fi
 
     for cellbase_record in cell_recs:
         n_ev_strings_per_record = 0
+        a_dictionary = cellbase_record if "clinvarSet" not in  cellbase_record else cellbase_record['clinvarSet']
```

Comment:
Sure, I didn't name this, but I didn't rename it during refactoring either. I'll rename both this and the parameter.